src/js/index.js

```diff
@@ -31,14 +30,17 @@ const chainId = 5;
 // const contractAddress = '0x6Fc000Ba711d333427670482853A4604A3Bc0E03';
 
 // Contract with min bid of 0.001 ETH and threshold of 0.1 ETH
-const contractAddress = '0xb96a231384eeea72a0edf8b2e896fa4bacaa22ff';
+const contractAddress = '0xb96A231384eEeA72A0EDF8b2e896FA4BaCAa22fF';
 
 
 /////////////
 // Web3Modal Things
 /////////////
 
-const {publicClient} = configureChains(chains, [w3mProvider({projectId})])
+const {publicClient} = configureChains(chains, [
+    // w3mProvider({projectId}),
+    infuraProvider({apiKey: '2096b0699ab146b1a019961a2a9f9127'})])
```

Comment:
I suppose we can use IP address permissions to prevent this from being leaked? or an environment variable?